starwars/src/components/CharacterList.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import Character from './Character';
+import './StarWars.css'
+
+const CharacterList = props => {
+return <div className = "character-container"> {props.starwarsChars.map((character)=>{
+    return (
```

Comment:
in the console, you will be getting an error asking for a key remember when doing a map the 1st object needs to have a unique key